README.md

```diff
@@ -15,7 +15,10 @@ for the following use cases:
 * Workload debugging
 * Custom data collection via extensible plugins
 
-Sonobuoy supports 3 Kubernetes minor versions: the current release and 2 minor versions before. Sonobuoy is currently versioned to track the Kubernetes minor version to clarify the support matrix. For example, Sonobuoy v0.14.x would support Kubernetes 1.14.x, 1.13.x, and 1.12.x.
+Starting v0.20, Sonobuoy supports Kubernetes v1.17 or earlier.
+Sonobuoy no longer follows Kubernetes release cycle -- this means that Sonobuoy releases in the future may not follow the same cadence as Kubernetes releases.
```

Comment:
Probably just say that Sonobuoy releases will be independent and not necessarily coincide with a Kubernetes release.